router.go

```diff
@@ -187,6 +188,27 @@ func (p *ControllerRegister) addWithMethodParams(pattern string, c ControllerInt
 	route.methods = methods
 	route.routerType = routerTypeBeego
 	route.controllerType = t
+	route.initialize = func() ControllerInterface {
```

Comment:
we can, but I don't think we want.
having initialize set to nil doesn't seem like an expected behavior.
also the only way (I see) to find out if there are any init parameters, is to run the code that is already in the initialize function.